cmd/bucket-handlers.go

```diff
@@ -66,9 +66,9 @@ func initFederatorBackend(objLayer ObjectLayer) {
 				}
 				return gerr
 			}
-			if r.Host != globalDomainIP {
+			if intersection := globalDomainIPs.Intersection(set.CreateStringSet(r.Hosts...)); intersection.IsEmpty() {
```

Comment:
No need for a new variable you can simply use... `globalDomainIPs.Intersection(set.CreateStringSet(r.Hosts...)).IsEmpty()` ?